src/tic_tac_toe/player.clj

```diff
@@ -0,0 +1,12 @@
+(ns tic-tac-toe.player)
+
+(def human "X")
+(def computer "O")
+
+(defn random-move []
```

Comment:
If this feature isn't available for playing then I would get rid of it. But if you wanted to add the ability to play an easy computer player then it might be worth keeping